homeassistant/components/iron_os/quality_scale.yaml

```diff
@@ -0,0 +1,82 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: Integration does not have actions
+  appropriate-polling: done
```

Comment:
I think 5 seconds is the lowest interval possible, is there a reason for this?

Reply:
yes, the temperature changes are pretty fast and as the device is only connected when used, it won't be filling up the state machine with values 24/7. Automating a fume extractor to turn on when you start soldering wouldn't be possible otherwise. This fast interval is really only for the live data, which is read in bulk from a single characteristic, I want to add more entities, which are on different characteristics, these will of course have a different polling interval of 60s.